soc/st/stm32/stm32f2/Kconfig

```diff
@@ -1,16 +1,13 @@
-# ST Microelectronics STM32F2X MCU series
+# STMicroelectronics STM32F2X MCU series
 
 # Copyright (c) 2018 qianfan Zhao <qianfanguijin@163.com>
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_STM32F2X
-	bool "STM32F2x Series MCU"
 	select ARM
 	select CPU_CORTEX_M3
 	select CPU_CORTEX_M_HAS_DWT
 	select SOC_FAMILY_STM32
```

Comment:
Remove `select SOC_FAMILY_STM32`

Reply:
done

Reply:
Not done. This is the cause of the dependency loop reported in CI.